textfsm/clitable.py

```diff
@@ -24,6 +24,9 @@
 Is the glue between an automated command scraping program (such as RANCID) and
 the TextFSM output parser.
 """
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
```

Comment:
Last two lines are not strictly needed but thinking it might be prudent to include them anyway.